es6-lib/util/metrics.js

```diff
@@ -9,12 +9,59 @@ var conf = config();
 
 class Metrics {
   constructor() {
-    setInterval(this._sampleMemory.bind(this), 1000);
+    this._resetState();
     this._gc = gcstats();
     this._gc.on('stats', _.throttle(this._onGcStat, 1000).bind(this));
```

Comment:
Prevents thrashing by calling the `this._onGcStat ` function no more than once every 1,000ms.